components/mpas-ocean/cime_config/config_component.xml

```diff
@@ -122,7 +122,7 @@
  </entry>
 
   <description>
-    <desc compset="_MPASO">mpas ocean default:</desc>
+    <desc compset="_MPASO">MPAS ocean default</desc>
```

Comment:
I think the colon is consistent with the other components - why are we removing it with this PR?

Reply:
Jon,

here is how the output looks right now:
...
ATM component is Data driven ATM interannual JRA55 forcing
LND component is Stub land component
ICE component is MPAS seaice
OCN component is MPAS ocean default
ROF component is Data runoff modelJRA55 interannual forcing
GLC component is Stub glacier (land ice) component
WAV component is Stub wave component
ESP component is 
...

and how it looked before:
...
ATM component is  Data driven ATM interannual JRA55 forcing
LND component is Stub land component
ICE component is MPAS seaice:
OCN component is mpas ocean default:
ROF component is Data runoff modelJRA55 interannual forcing
GLC component is Stub glacier (land ice) component
WAV component is Stub wave component
ESP component is 
...

so, no, the colon in not consistent with the other components and neither is the extra space ;-)

Reply:
OK, good catch then -- and thanks! But please separate this into two PR's then -- one to get the ARRM configurations working and one to clean up output